examples/compute/windowsvirtualmachine.yaml

```
@@ -2,26 +2,118 @@ apiVersion: compute.azure.upbound.io/v1beta1
 metadata:
   annotations:
-    upjet.upbound.io/manual-intervention: "This resource requires manual intervention."
+    meta.upbound.io/example-id: compute/v1beta1/windowsvirtualmachine
+  labels:
+    testing.upbound.io/example-name: example
   name: example
 spec:
   forProvider:
     adminPasswordSecretRef:
-      key: example-key
+      key: password
       name: example-secret
-      namespace: crossplane-system
+      namespace: upbound-system
     adminUsername: adminuser
     location: West Europe
-    networkInterfaceIds:
-    - '...'
+    networkInterfaceIdsRefs:
+    - name: example
     osDisk:
     - caching: ReadWrite
       storageAccountType: Standard_LRS
-    resourceGroupNameRef:
-      name: example-compute
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
     size: Standard_F2
     sourceImageReference:
     - offer: WindowsServer
       publisher: MicrosoftWindowsServer
       sku: 2016-Datacenter
       version: latest
+
+---
+
+apiVersion: network.azure.upbound.io/v1beta1
+kind: NetworkInterface
+metadata:
+  annotations:
+    meta.upbound.io/example-id: compute/v1beta1/windowsvirtualmachine
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    ipConfiguration:
+    - name: internal
+      privateIpAddressAllocation: Dynamic
+      subnetIdSelector:
+        matchLabels:
+          testing.upbound.io/example-name: example
+    location: West Europe
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
+
+apiVersion: azure.upbound.io/v1beta1
+kind: ResourceGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: compute/v1beta1/windowsvirtualmachine
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    location: West Europe
+
+---
+
+apiVersion: network.azure.upbound.io/v1beta1
+kind: Subnet
+metadata:
+  annotations:
+    meta.upbound.io/example-id: compute/v1beta1/windowsvirtualmachine
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    addressPrefixes:
+    - 10.0.2.0/24
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    virtualNetworkNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
+
+apiVersion: network.azure.upbound.io/v1beta1
+kind: VirtualNetwork
+metadata:
+  annotations:
+    meta.upbound.io/example-id: compute/v1beta1/windowsvirtualmachine
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    addressSpace:
+    - 10.0.0.0/16
+    location: West Europe
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+    meta.upbound.io/example-id: compute/v1beta1/windowsvirtualmachinescaleset
```

```suggestion
    meta.upbound.io/example-id: compute/v1beta1/windowsvirtualmachine
```
Could you please fix example-id?

---

fixed